src/index.html

```diff
@@ -2,15 +2,55 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+     method="post"
+     action="#"
+     data-qa="big"
+     autocomplete="off"
+     class="search_form"
+     >
+      <img
+        src="images/Search.svg"
+        alt="icon"
+        class="search_icon"
+       >
+      <label for="big">
+        <input
+        id ="big"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        type="search"
+        class="search-input"
+      >
```

Comment:
Same here. Please, check it everywhere. Go through the checklist one more time
```suggestion
        <input
          id ="big"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
          type="search"
          class="search-input"
        >
```